src/constants/index.js

```diff
@@ -0,0 +1,13 @@
+const baseUrls = {
+    development:{
+        development: 'http://localhost:8000',
+    },
+    test:{
```

Comment:
Is line 5 `NODE_ENV` and line 6 `REACT_APP_ENVIRONMENT`? Meaning that we could include in the future another test environment, e.g. staging, as shown below?
```
test:{
        development: 'http://localhost:8000',
        staging: 'https://staging.trytechlab.com/diana',
},
```
Or even production -> disaster recovery?
```
production:{
        production: 'https://trytechlab.com/diana',
        disasterRecovery: 'https://dr.trytechlab.com/diana',
},
```
If that is the case, I very much like it.

Reply:
Exactly that! Until I've done this task, I knew only that we need a dev and a prod configurations, but once I've ran the tests it used a test environment. So I think that it's configurable, and we could do things like, run on a staging, qa, sandbox, disasterRecovery and so on. 🙂

Reply:
Nice!